index.bs

```diff
@@ -3199,6 +3203,129 @@ The [=remote end steps=] with |command parameters| are:
 1. Return a new map matching the <code>ScriptEvaluateResultSuccess</code>
    production, with the <code>result</code> field set to |result|.
 
+#### The script.callFunction Command ####  {#command-script-callFunction}
+
+The <dfn export for=commands>script.callFunction</dfn> command calls a provided
+function with given arguments in a given realm.
+
+ <code>RealmInfo</code> can be either a realm or a browsing context.
+
+Issue: declare `RemoteValueArgument`.
+
+Issue: declare `LocalValueArgument`.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      ScriptCallFunctionCommand = {
+        method: "script.callFunction",
+        params: ScriptCallFunctionParameters
+      }
+
+      ScriptCallFunctionParameters = {
+        functionDeclaration: text;
+        arguments: [CallFunctionArgument];
+        ?awaitPromise: bool;
+        target: Target;
+      }
+
+      CallFunctionArgument = (
+        RemoteValueArgument //
+        LocalValueArgument
+      );
+
+      RemoteValueArgument = {
+        objectId: string;
+      };
+
+      LocalValueArgument = {
+        type: string;
```

Comment:
Presumably "type" is going to be a closed set of values that we know how to deserialize.